posts/classlist.md

```diff
@@ -2,6 +2,6 @@ feature: classList
 status: use
 tags: polyfill gtie9 nooldmobile
 kind: api
-polyfillurls:[classlist.js](https://github.com/eligrey/classList.js)
+polyfillurls: [classlist.js](https://github.com/eligrey/classList.js)
 
-classList is a relatively new feature which may not be available in older browsers and Internet Explorer <= 9. While this is not a concern if you are using a library like jQuery to modify class names, make sure you use the polyfill when using classList directly.
+`classList` is well supported nowadays, but expect some quirks in IE10-11 and Opera Mobile, like missing methods or different method signatures (see [caniuse notes](https://caniuse.com/#feat=classlist)).
```

Comment:
technically i'm not seeing "missing methods" in these IE/Opera issues... i guess i'd summarize it as "incomplete support" 🤷‍♂️